src/components/ui/tabs.tsx

```diff
@@ -1,12 +1,14 @@
-import { cn } from "@/lib/utils";
+import { forwardRef, type ElementRef, type ComponentPropsWithoutRef } from "react";
 import * as TabsPrimitive from "@radix-ui/react-tabs";
-import * as React from "react";
 
+import { cn } from "@/lib/utils";
+
+// FIXME: Unused?
```

Comment:
Components inside `ui` folder automatically generated from [shadcn/ui](https://ui.shadcn.com), so this comment here is unrelated, due I do not maintain this part of code.